skrf/network.py

```diff
@@ -4052,6 +4052,34 @@ def average(list_of_networks: Sequence[Network], polar: bool = False) -> Network
     return out_ntwk
 
 
+def stdev(list_of_networks: Sequence[Network], attr: str = 's_db') -> npy.ndarray:
```

Comment:
Is it a good idea, to let users calculate statistics on logarithmic values like `s_db`?
mean(s_db) differs from 20 * log10(mean(s)).

Reply:
It's not, that was something I forgot to change. Thank you for pointing this out. I think defaulting to 's' instead of 's_db' is a safer practice for now.